### PR TITLE
DF-1998 removed reference service 2

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/shared/backend-services/reference-service/occupation-label.repository.ts [hunk 1]
```diff
@@ -1,44 +1,25 @@
...
-} from 'rxjs';
+import { EMPTY, Observable } from 'rxjs';
 import {
   OccupationLabelAutocomplete,
   OccupationLabelData
 } from './occupation-label.types';
-import {
-  catchError,
-  shareReplay,
-  map
```
Comment (hamidsoleymani) [inline alv-portal-ui/src/app/shared/backend-services/reference-service/occupation-label.repository.ts]: map is gone?

Comment (ganqqwerty) [reply alv-portal-ui/src/app/shared/backend-services/reference-service/occupation-label.repository.ts]: I will bring it back

File: alv-portal-ui/proxy.conf.staging.json [hunk 2]
```diff
@@ -34,11 +34,6 @@
     "secure": false,
     "loglevel": "debug"
   },
-  "/referenceservice-2/*": {
```
Comment (cjpsycos) [inline alv-portal-ui/proxy.conf.staging.json]: what is about other environment?

Comment (ganqqwerty) [reply alv-portal-ui/proxy.conf.staging.json]: it doesn't exist in other envs

**Pull request merged**